index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TanmayiPortfolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <!--Outer div consists toolbar and headline-->
+    <div class="wrapper">
+        <!--section one toolbar-->
+        <header class="toolbar">
+          <div class="statusbar">
+            <div class="time">
+                <img src="./media/TimeStyle.svg" alt="Time stamp">
+            </div>
+            <div class="bars">
+                <img src="./media/CellularConnection.svg" alt="signal bar">
+                <img src="./media/Wifi.svg" alt="wifi bar">
+                <img src="./media/Battery.svg" alt="battery">
+            </div>
+          </div>
+          <nav class="logo-hamburger">
+            <a href='#' class="logo"><span>T</span></a>
+            <a href='#' class="hamburger"><img src="./media/hamburger.png" alt="menu"></a>
+          </nav>
+        </header>
+        <!--section two headline-->
+        <section class="headline">
+          <div class="primary text">
+            <p>Hey there.<br>
+                I'm Tanmayi.<br>
+            </p>
+            <p id="green">I'm a software developer</p>
```

Comment:
:red_circle: I think this is a subtitle. The semantic tag here should be a header tag hx (examples:` <h1>,<h2>,<h3>, `etc...). This way will be semantically correct and the screen readers and other technologies will thank you.